docs/usage.md

```diff
@@ -53,12 +53,13 @@ them.
 ### Copying `default_config.yaml`
 
 If you have used Git to clone the TopoStats repository from GitHub the default configuration can be found in the
-sub-directory `topostats/default_config.yaml`. If you have installed TopoStats from PyPI then a sample configuration
+sub-directory `topostats/default_config.yaml`. 
+
+If you have installed TopoStats from PyPI then a sample configuration
 file can be downloaded from
-[here](https://raw.githubusercontent.com/AFM-SPM/TopoStats/main/topostats/default_config.yaml) (right-click on the link
-and select `Save As` to save the file to your computer).
+[here](https://github.com/AFM-SPM/TopoStats/blob/a180198b369e68dd892038dca1893aa396b04d33/topostats/default_config.yaml), or copy and paste the contents into a text file ending with `.yaml`.
```

Comment:
As above, possibly confusion/challenges with people saving files with a `.yaml` extension, saving is straight-forward.